app/assets/javascripts/discourse/app/templates/components/invite-link-panel.hbs

```diff
@@ -0,0 +1,59 @@
+{{#if inviteModel.error}}
+  <div class="alert alert-error">
+    {{html-safe errorMessage}}
+  </div>
+{{/if}}
+
+<div class="body">
+  {{#if inviteModel.finished}}
+    {{generated-invite-link link=inviteModel.inviteLink}}
+  {{else}}
+
+    <div class="group-access-control">
+      <label class="instructions">
+        {{i18n "topic.automatically_add_to_groups"}}
+      </label>
+      {{group-selector
+        fullWidthWrap=true
+        groupFinder=groupFinder
+        groupNames=inviteModel.groupNames
+        placeholderKey="topic.invite_private.group_name"}}
+    </div>
+
+    <div class="max-redemptions-allowed">
+      <label class="instructions">
+        {{i18n 'user.invited.invite_link.max_redemptions_allowed_label'}}
+      </label>
+      {{input type="number" value=maxRedemptionAllowed class="max-redemptions-allowed-input" min="2" max="1000000"}}
```

Comment:
Should we use the site settings for `1000000` instead?

Reply:
We should, done.